pico/builder/tests/builder/src/test/java/io/helidon/pico/builder/api/test/MyConfigBeanTest.java

```diff
@@ -35,47 +35,47 @@ void manual() {
                    equalTo("MyConfigBeanManualImpl(name=null, enabled=false, port=0)"));
 
         val = MyConfigBeanManualImpl.toBuilder(val)
-                .name("jeff")
+                .name("test")
                 .enabled(true)
                 .port(80)
                 .build();
-        assertThat(val.toString(), equalTo("MyConfigBeanManualImpl(name=jeff, enabled=true, port=80)"));
+        assertThat(val.toString(), equalTo("MyConfigBeanManualImpl(name=test, enabled=true, port=80)"));
```

Comment:
Doing assertions on `toString()` is not a good practice, as you make the `toString()` result a specification, where it should be a human readable string (but not specified or something you should ever depend on).
If you want to test the builder contains the correct values, use getters.
If you want to test the `toString()` contains all values, use multiple assertions with contains string (e.g. look just for the value, not for the whole format of toString())

TL;DR: do not test toString(), we may want to change it and it should not result in a failed test

Reply:
The comment above (assertion on `toString()` is applicable to multiple tests)

Reply:
discussed this w/ Tomas. marking resolved for now - will create a task to do this later.